plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -114,6 +114,13 @@ $font-sf-pro-display: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe U
 		}
 	}
 
+	.other-payment-gateways__header__title__image-placeholder {
```

Comment:
@dmallory42 what is this about? I guess we can remove it, I don't see where it's used

Reply:
I see where it's used!